docker-compose.yml

```diff
@@ -1,29 +1,56 @@
 version: '3.7'
 
+x-shared_environment: &shared_environment
+  LOG_LEVEL: ${LOG_LEVEL}
+  DATABASE_HOST: db
+  DATABASE_NAME: ${DATABASE_NAME:-vapor_database}
+  DATABASE_USERNAME: ${DATABASE_USERNAME:-vapor_username}
+  DATABASE_PASSWORD: ${DATABASE_PASSWORD:-vapor_password}
+
 services:
   {{name}}:
     image: {{name}}:latest
     build:
       context: .
+    environment:
+      <<: *shared_environment
     {{#fluent}}depends_on:
       - db{{/fluent}}
     ports:
       - '8080:80'
 
+  {{#fluent}}migrator:
+    image: {{name}}:latest
+    build:
+      context: .
+    environment:
+      <<: *shared_environment
+    depends_on:
+      - db
+    entrypoint:
+      - '/bin/bash'
+    command:
+      - '-c'
+      - "echo 'y' | ./Run migrate"
```

Comment:
This service could take advantage of the `vapor-entrypoint.sh` like the primary service if we added a `-y` or `--yes` or `--no-prompt` or some such named option to the `migrate` command. The only reason it got a bit messier here is so that I can pass "y" into the `migrate` command to force the migrator service to _just go_.

Reply:
ConsoleKit (which the migrate command is built on) should support passing `--yes` or `-y` to bypass any confirmations.

Reply:
Hell yeah. Cleans up nicely: https://github.com/vapor/template/pull/7/commits/f5dcf9d97ac000b2231707dbb0369197b379d5e4